website/server/controllers/api-v3/content.js

```diff
@@ -134,4 +143,49 @@ api.getContent = {
   },
 };
 
+/**
+ * @api {get} /api/v3/content/hallOfHeroes Get hall of heroes item path gifting reference.
+ * @apiDescription Does not require authentication.
+ * @apiName getAllOfHeroesItemReference
+ * @apiGroup Content
+ *
+ * @apiSuccess {Object} Formatted hall of heroes gifting item reference
+ */
+api.getHallOfHeroesItemReference = {
+  method: 'GET',
+  url: '/content/hallOfHeroes',
+  async handler (req, res) {
+    let content = _.cloneDeep(common.content);
```

Comment:
Also, what's the reason for cloning it? Looks like everywhere you use it, you call `_.keys` which returns a new Array. There's no danger of mutating the content file.